aqua/LyX/Portfile

```diff
@@ -21,14 +21,14 @@ long_description    LyX is an advanced open source document processor \
 
 platforms           darwin
 
-depends_build-append    port:bison port:gawk port:cctools \
-                        port:aspell port:hunspell
```

Comment:
Is it intentional you have removed hunspell here, but not added it to depends_lib below ? Even if it is implicitly depended on, via another port, it should be explicitly listed as this port directly links to it.

Reply:
Sorry. Forget this as I am clearly blind...